content/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -0,0 +1,109 @@
+---
+title: Menggunakan kubectl untuk membuat Deployment
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+         <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Belajar tentang aplikasi Deployment.</li>
+                    <li><i>Deploy</i> aplikasi pertama kamu di Kubernetes dengan kubectl.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3><i>Deployment</i> Kubernetes</h3>
+                <p>
+                Ketika klaster Kubernetes kamu sudah berjalan, kamu dapat men-<i>deploy</i> aplikasi terkontainerisasi di atasnya.
```

Comment:
can we use the Indonesian word "menyebarkan"?
```suggestion
                Ketika klaster Kubernetes kamu sudah berjalan, kamu dapat menyebarkan aplikasi terkontainerisasi di atasnya.
```

Reply:
I'm not sure if people generally understand "menyebarkan" for deployment

Reply:
Any suggestions for this? "menjalankan" is already used by other word ("run"), deploy & run are different by definition 🤔

Reply:
How about "menggelar"? https://kbbi.kemdikbud.go.id/entri/menggelar